lib/avromatic/model/serialization.rb

```diff
@@ -41,7 +41,7 @@ def avro_hash(fields)
             result[key.to_s] = if value.is_a?(Avromatic::Model::Attributes)
                                  value.value_attributes_for_avro
                                else
-                                 value
+                                 avro_coder[key].try(:call, value) || value
```

Comment:
Should we give the `avro_coder` a default no-op coder so we can avoid the `try`?